boards/st/nucleo_h563zi/doc/index.rst

```diff
@@ -320,3 +343,6 @@ You can debug an application in the usual way.  Here is an example for the
 
 .. _STM32CubeProgrammer:
    https://www.st.com/en/development-tools/stm32cubeprog.html
+
+.. _OpenOCD official Github mirror:
+   https://github.com/openocd-org/openocd/commit/34afaa939d5d7512efc37110d0a266ea9004b15c
```

Comment:
This link is not valid

Reply:
replaced by https://github.com/openocd-org/openocd/